vimeo-networking/src/main/java/com/vimeo/networking/VimeoClient.java

```diff
@@ -491,6 +491,40 @@ public VimeoAccount authorizeWithClientCredentialsGrantSync() {
         return call;
     }
 
+    /**
+     * Register the user using Google authentication token.
+     *
+     * @param googleToken {@code id_token} value received by Google after authenticating.
+     * @param email User email address.
+     * @param callback This callback will be executed after the request succeeds or fails.
+     * @return a retrofit {@link Call} object, which <b>has already been enqueued</b>.
+     */
+    @Nullable
+    public Call<VimeoAccount> joinWithGoogleToken(String googleToken, String email, AuthCallback callback) {
+        if (callback == null) {
+            throw new AssertionError("Callback cannot be null");
+        }
+
+        if (googleToken == null || googleToken.isEmpty()) {
+            final VimeoError error = new VimeoError("Google authentication error.");
+
+            if (googleToken == null || googleToken.isEmpty()) {
```

Comment:
Good catch.  I used the `loginWithFacebookToken`  / `joinWithFacebookToken` methods as a starting point for these and they had the same issue.  Fixed in all 4 places.